components/network/fail2ban/Makefile

```diff
@@ -1,26 +1,33 @@
 #
-# This file and its contents are supplied under the terms of the
-# Common Development and Distribution License ("CDDL"), version 1.0.
-# You may only use this file in accordance with the terms of version
-# 1.0 of the CDDL.
+# CDDL HEADER START
```

Comment:
Why this header change?  We prefer the previous one.  See https://github.com/OpenIndiana/oi-userland/tree/oi/hipster/templates